src/adapters/firebase.js

```diff
@@ -0,0 +1,17 @@
+export default class FirebaseAuth{
+  constructor(firebase){
+    this.firebase = firebase;
+  }
+
+  authenticate(){
+    var provider = new this.firebase.auth.GithubAuthProvider();
```

Comment:
This repo is written in ES6 and transpiled to ES5 - code style should be consistent. Ie: `var` -> `let`/`const`, anon functions -> arrow functions, spacing between {...} tokens, etc.